src/PageContent/Calendar/DelivererGroupContent.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react';
+import { AccountType, StringFormat } from '../../Enums';
+import './Content.css';
+import volunteer from '../../icons/volunteer.svg';
+
+class DelivererGroupContent extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            edit: false,
+            delivererGroup: this.props.delivererGroup,
+            deliverer1: this.props.delivery.delivererGroup.deliverers[0].name,
+            deliverer2: this.props.delivery.delivererGroup.deliverers[1].name,
+            phone1: this.props.delivery.delivererGroup.deliverers[0].phone,
+            phone2: this.props.delivery.delivererGroup.deliverers[1].phone,
+            email1: this.props.delivery.delivererGroup.deliverers[0].email,
+            email2: this.props.delivery.delivererGroup.deliverers[1].email
+        };
+        this.edit = this.edit.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+    }
+    edit() {
+        this.setState({
+            edit: true
+        });
+    }
+
+    handleChange(e) {
+        e.preventDefault();
+
+        this.setState({
+            edit: false,
+            deliverer1: e.target.deliverer1.value,
+            phone1: e.target.phone1.value,
+            deliverer2: e.target.deliverer2.value,
+            phone2: e.target.phone2.value
+        });
+
+        // TODO: Save values to firebase
+    }
+
+    render() {
+        return (
+            <div className="wrapper">
+                <img className="content-icon" src={volunteer} alt="volunteer" />
+                <div className="content-wrapper">
+                    {this.props.accountType === AccountType.DELIVERER_GROUP ? (
+                        <h1 className="section-header">Student Deliverers</h1>
+                    ) : (
+                        <h1 className="section-header">Picking Up Donation</h1>
+                    )}
+                    <h2 className="organization">
+                        {this.props.delivererGroup}
+                    </h2>
+                    {!this.state.edit ? (
```

Comment:
Need to add another condition block to check whether `this.state.deliverers` is empty or not. 
If it is empty (`this.state.deliverers.length === 0`):
- if account is DG, i think maybe the big red `Assign Volunteers` button like in figma? and it'll act the same as the `Edit` button?
- if not DG, maybe just a text saying something like "TBD"?